packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -103,7 +103,6 @@ library TaikoData {
         uint96 contestBond;
         uint64 timestamp; // slot 6 (90 bits)
         uint16 tier;
-        uint8 contestations;
```

Comment:
@davidtaikocha notice this change and let me know if it's a problem for node/client.

Reply:
Its ok on client side